src/func.jl

```diff
@@ -28,6 +30,7 @@ mutable struct FunctionOperator{iip,oop,T<:Number,F,Fa,Fi,Fai,Tr,P,Tt,C} <: Abst
                               traits,
                               p,
                               t,
+                              kwargs_for_op,
```

Comment:
I think you can stick with the same name `accepted_kwargs`.

Reply:
good point, fixed